code/game/objects/items/mountable_frames/paintings.dm

```diff
@@ -332,9 +384,14 @@ var/global/list/available_paintings = list(
 
 
 /obj/structure/painting/cultify()
-	var/obj/structure/painting/narsie/N = new(loc)
-	N.pixel_x = pixel_x
-	N.pixel_y = pixel_y
+	if(icon_state == "deer")
```

Comment:
Hardcoded because making the new paintings doesn't make them into a subtype, hence the big switch statement of icon_state a bit above